Assignments/Sorting/SelectionSort.playground/Pages/Selection Sort.xcplaygroundpage/Contents.swift

```diff
@@ -9,19 +9,32 @@ Better define an inner function to find the next minimum in the array.
 
 */
 
-func selectionSort(array: [Int]) -> [Int] {
+func selectionSort(var array: [Int]) -> [Int] {
     // You may declare array argument with var keyword so that it is copied.
+    
+    for j in array.startIndex..<array.endIndex - 1 {
```

Comment:
I generally lean toward using start/endIndex for future-proofing, I guess. In this case we know it's an `Array` and therefore its index is `Int`, so it's unnecessary, but I've gotten into the habit of using the methods on the protocol (`CollectionType`) when I can.